app/BUILD.bazel

```diff
@@ -315,23 +317,13 @@ kt_android_library(
     enable_data_binding = 1,
     manifest = "src/main/ViewModelManifest.xml",
     deps = [
-        ":dagger",
+        "//domain",
         "//model",
         "//utility",
-        "//domain",
-        ":listeners",
         ":annotations",
```

Comment:
Deps ordering is off here & elsewhere: local package deps should come before top-level package deps.

Reply:
Fixed